### PR TITLE
ci: Integration test

### PR DESCRIPTION
**Pull request by david-doit-intl**
Signed-off-by: david-doit-intl <david@doit-intl.com>

### CODE REVIEW

**Review by stepanstipl — COMMENTED** [review]
This looks cool. I'm wondering if we could easily make this runnable locally? I.e. something like a make task. Not sure about the kind part though, but in general I would prefer to have parity between tests runnable in CI and locally.

Minor note: the commits should be probably prefixed with`ci:`, (and/or `build:` if we touch the make/local part). The `feat:` should be reserved for app features.

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -55,10 +57,45 @@ jobs:
...
+        path: release-artifacts
+    - name: Create k8s Kind Cluster
+      uses: helm/kind-action@v1.1.0
+      with:
+        node_image: ${{ matrix.k8s_version }}
+        cluster_name: kubent-test-cluster
+    - name: run integration test
+      run: |
+        tar xvzf release-artifacts/kubent-${{ github.event.number }}merge-linux-amd64.tar.gz
+        kubectl version --short
```
[the remark this answers is not in the document]
Comment (david-doit-intl) [reply .github/workflows/main.yaml:93]: yes the intent is to run kubent, I haven't finished it yet :)

**Comment by david-doit-intl** [comment]
> This looks cool. I'm wondering if we could easily make this runnable locally? I.e. something like a make task. Not sure about the kind part though, but in general I would prefer to have parity between tests runnable in CI and locally.
> 
> Minor note: the commits should be probably prefixed with`ci:`, (and/or `build:` if we touch the make/local part). The `feat:` should be reserved for app features.

@stepanstipl please have a look, I am running kubent against all the k8s versions (there is nothing in the cluster however) I'll fix this shortly or in annother pr? Your thoughts would be appreicated

**Review by stepanstipl — COMMENTED** [review]
Yep, I think this is good for now 👍 , we can look into adding further actual tests later on. This is already valuable, as it shows that `kubent` can run against given K8s version. 🎉 

Just the commit msg. please.

And maybe we can make the build also depend on `pre-commit`? I think it's the fastest test, and doesn't really make sense to start any other, more expensive ones, unless that one is sorted WDYT?

Screenshot 2021-06-10 at 14 33 49

**Comment by david-doit-intl** [comment]
Agreed let me make those changes!

**Comment by david-doit-intl** [comment]
@stepanstipl done.

**Review by stepanstipl — APPROVED** [review]
Cool, LGTM! Thanks 😎

**Pull request merged**